DigitalLearningSolutions.Data.Migrations/RecurringSqlJobs/DeleteEmailVerificationHashesRecordsOlderThan4Days.sql

```diff
@@ -0,0 +1,5 @@
+﻿-- Deletes EmailVerificationHashes records older than 4 days. Should be run every night at 2:02AM.
+BEGIN
```

Comment:
We are deleting EmailVerificationHashes that are more than 4 days old every day and checking elsewhere in the code that the hash created date isn't more than 14 days old?